src/utils.h

```diff
@@ -65,6 +65,12 @@
 #define STRINGIFY(x) #x
 #define STRINGIFY2(x) STRINGIFY(x)
 
+/* Check if a char is valid in a string literal */
+static inline bool
+is_valid_char(char c) {
```

Comment:
The style (though I don't really like it myself...) is to put the `{` of functions in its own line

Reply:
Sorry